core/kazoo_proper/src/pqc_cb_accounts.erl

```diff
@@ -139,3 +144,54 @@ postcondition(Model
                  ),
             500 =:= pqc_cb_response:error_code(APIResult)
     end.
+
+-spec seq() -> 'ok'.
+seq() ->
+    enable_and_delete_topup().
+
+-spec enable_and_delete_topup() -> 'ok'.
+enable_and_delete_topup() ->
+    API = pqc_cb_api:init_api(['crossbar'], ['cb_accounts']),
+
+    %% Make sure everything is clean for the test.
+    cleanup(API),
+
+    AccountResp = create_account(API, hd(?ACCOUNT_NAMES)),
+    ?INFO("created account: ~s", [AccountResp]),
+
+    AccountJObj = kz_json:get_value(<<"data">>, kz_json:decode(AccountResp)),
+    AccountId = kz_json:get_binary_value(<<"id">>, AccountJObj),
+    TopupConfig = kz_json:from_list([{<<"threshold">>,10},{<<"amount">>,50}]),
+    RequestData = kz_json:set_value(<<"topup">>, TopupConfig, AccountJObj),
+    RequestEnvelope = pqc_cb_api:create_envelope(RequestData),
+
+    Resp = topup_request(AccountId, API, RequestEnvelope),
+    ?INFO("enable topup resp: ~s", [Resp]),
+
+    RespJObj = kz_json:get_value(<<"data">>, kz_json:decode(Resp)),
+    'true' = kz_json:are_equal(TopupConfig, kz_json:get_ne_value(<<"topup">>, RespJObj)),
+    RequestData1 = kz_json:delete_key(<<"topup">>, RespJObj),
+    RequestEnvelope1 = pqc_cb_api:create_envelope(RequestData1),
+
+    Resp1 = topup_request(AccountId, API, RequestEnvelope1),
+    ?INFO("disable topup resp: ~s", [Resp1]),
+
+    'undefined' = kz_json:get_ne_value(<<"topup">>, kz_json:decode(Resp1)),
+
+    cleanup(API),
+    ?INFO("FINISHED ENABLE AND DISABLE TOPUP CHECKS").
+
+-spec cleanup(pqc_cb_api:state()) -> any().
+cleanup(API) ->
+    ?INFO("CLEANUP TIME, EVERYBODY HELPS"),
+    _ = cleanup_accounts(API, ?ACCOUNT_NAMES),
+    _ = pqc_cb_api:cleanup(API).
+
+-spec topup_request(kz_term:ne_binary(), pqc_cb_api:state(), kz_json:object()) -> pqc_cb_api:response().
+topup_request(AccountId, API, RequestEnvelope) ->
```

Comment:
Functions to call the API server should have the `API` data in the first parameter - this will become important when we extend the proper-testing to this API function.